### ISSUE TITLE
[Improvement] Structures for every DBC File

### ISSUE THREAD

**Issue by Dokman** [callmephil/LazyWorker#2]
If we use a Structure option we can handle all the languages, we know it's a difficult option but we can hardcode a lot of things and then add the custom rest and it'sa lot better and we don't have to fuck our minds with hundred of lines of DBC

**Comment by callmephil** [comment]
thanks i will consider this option. for now i'm coding functional. then i will try to improve coding style.

**Comment by Dokman** [comment]
i can do it if you want :) i can prepare the structures and make it usable :)

**Comment by callmephil** [comment]
Yes why not. but i'm coding new features and adding more details to older ones so UI and code may change a lot during this week. it's preferable to have a "stable" version before re-coding in structures. add me on skype if you want Philippe19-94

**Comment by Dokman** [comment]
i added you, sorry i was late to see the message :(
